src/test/java/org/commcare/formplayer/tests/RestoreFactoryTest.java

```diff
@@ -149,7 +149,7 @@ public void testNotIsRestoreXmlExpiredWeekly() {
     public void testGetCaseRestoreUrl() {
         restoreFactorySpy.setCaseId("case_id_123");
         assertEquals(
-                "http://localhost:8000/a/restore-domain/case_migrations/restore/case_id_123/",
+                "http://localhost:8000/a/restore-domain/phone/case_restore/case_id_123/",
```

Comment:
There are a few similar `/case_migrations/restore/` URLs in this file - shouldn't those also be updated? Interesting that those tests pass.

Reply:
Thanks Jenny! Fixed here: 4620913c8a86ad86f4b4b115d9196aea3a874f86
I guess those tests passed because my branch to remove the `case_migrations` app in commcarehq is not merged into master yet.